<launch> 
    <!-- Blood Oxigenation Measurement Sensor -->    
    <node name="g3t1_1" pkg="component" type="g3t1_1" output="screen" />

    <param name="frequency" value="1.3" /> <!-- 0.2 Hz  -->

    <!-- Defines the percentages to consider low, moderate or high risk -->
    <param name="lowrisk" value="0,20" />
    <param name="midrisk" value="21,65" />
    <param name="highrisk" value="66,100" />

    <!-- Risk values for oximeter -->
    <param name="HighRisk0" value="-1,-1" />
    <param name="MidRisk0" value="-1,-1" />
    <param name="LowRisk" value="65,100" />
    <param name="MidRisk1" value="55,65" />
    <param name="HighRisk1" value="0,55" />

    <!-- accuracy in percentage -->
    <param name="accuracy" value="90" type="double" />
    <param name="property" value="p2"/>

</launch>